packages/core/src/html/NodeProcessor.js

```diff
@@ -823,8 +837,6 @@ class NodeProcessor {
 
     this.postProcessNode(node);
 
-    insertTemporaryStyles(node);
```

Comment:
hmm not sure if we should be removing this here right away.
compilation alone isn't enough to completely remove fouc